apps/vanilla/src/pages/blackjack/game/dice.ts

```diff
@@ -0,0 +1,43 @@
+namespace Game {
+
+	/**
+	 * Game dice.
+	 */
+	export class Dice implements ObserverCore.Observer<UI.ThrowButtonClickedEventData> {
+
+		private readonly minValue: number = 1;
+
+		private readonly maxValue: number;
+
+		private readonly diceThrownEventNotifier: ObserverCore.Notifier<Game.DiceThrownEventData> =
+			new ObserverCore.Notifier<Game.DiceThrownEventData>();
+
+		/**
+		 * @inheritdoc
+		 */
+		public readonly observersRegistrar: ObserverCore.ObserversRegistrar<DiceThrownEventData> = this.diceThrownEventNotifier;
+
+		public constructor(maxValue: number) {
+			this.maxValue = maxValue;
+		}
+
+		/**
+		 * Throw dice.
+		 */
+		public throw(): void {
+			const minCeiled = Math.ceil(this.minValue);
+			const maxFloored = Math.floor(this.maxValue);
+			const newDiceValue = Math.floor(Math.random() * (maxFloored - minCeiled + 1) + minCeiled);
+
+			this.diceThrownEventNotifier.notify(new Game.DiceThrownEventData(newDiceValue));
+		}
+
+		/**
+		 * @inheritdoc
+		 */
+		// eslint-disable-next-line @typescript-eslint/no-unused-vars
+		public update(_message: UI.ThrowButtonClickedEventData): void {
+			this.throw();
+		}
```

Comment:
I believe you're able to write this
```suggestion
		public update(): void {
			this.throw();
		}
```